src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -1341,6 +1350,16 @@ public static synchronized List<ServiceInterface> getServicesFromInterface(Class
     }
     return ret;
   }
+  
+  /**
+   * Because startYml is required to be a static variable, since it's needed "before"
+   * a runtime instance exists it will be null in json serialization.  This method is needed
+   * so we can serialize the data appropriately.

```

Comment:
Has this been verified with both gson and jackson? I'm not sure both will pick up on the static getter. Quite frankly I'm not sure either will get it right, we had a lot of issues with Jackson thinking things were getters so I might've disabled that

Reply:
verified it currently with jackson, was the case with gson as well..
in fact its the case we general serialization in Java ... probably not worth changing at this point as I'm sure other things will break